docs/serverless/endpoint-response-actions/response-actions.mdx

```diff
@@ -179,6 +179,20 @@ You can follow this with the `execute` response action to upload and run scripts
 The default file size maximum is 25 MB, configurable in `kibana.yml` with the `maxUploadResponseActionFileBytes` setting. You must enter the value in bytes (the maximum is `104857600` bytes, or 100 MB).
 </DocCallOut>
 
+### `scan`
+
+Scan a specific file or directory on the host for malware. The scan uses the <DocLink slug="/serverless/security/configure-endpoint-integration-policy" section="malware-protection">malware protection settings</DocLink> (such as **Detect** or **Prevent** options, or enabling the blocklist) as configured in the host's associated ((elastic-defend)) integration policy. Use these parameters:
```

Comment:
```suggestion
Scan a specific file or directory on the host for malware. The scan action uses the <DocLink slug="/serverless/security/configure-endpoint-integration-policy" section="malware-protection">malware protection settings</DocLink> (such as **Detect** or **Prevent** options, or enabling the blocklist) as configured in the host's associated ((elastic-defend)) integration policy. Use these parameters:
```

Reply:
It could also be `...host for malware. It uses the...` or `...host for malware. Scan uses the...`

Reply:
Good eye for conciseness. I think I like `This uses the...`. I'll push a commit that updates this in both serverless and ESS docs.

Reply:
Nice! Thanks @joepeeples